doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -3,191 +3,54 @@
 
 This section highlights new features in {PRODUCT} {PRODUCT_VERSION}.
 
-ifdef::DM[]
+== Ability to only enable decision management components and functionality
 
-== {PRODUCT} {ENTERPRISE_VERSION} Kogito decision services support
+You can now configure the deployment to only enable decision management related components and functionality so that jBPM and case management features are disabled.
```

Comment:
Shouldn't we be more specific?  This relates to the fact that we now only have one set of images/operator for both RHPAM and RHDM, so there a new property on the templates and operator so you can run those with only DM capabilities enabled.